nvrdftd-taipei.md

```diff
@@ -0,0 +1,10 @@
+# Taipei, Taiwan
+
+### Restaurants for Eaters
+
+### Attractions for Tourists
+
+- Taipei 101
+- National Palace Museum
```

Comment:
@nvrdftd this sounds like a great place! Are there long lines to visit? Should I get a reservation?